gilbreth_perception/src/recognition_node.cpp

```diff
@@ -384,6 +387,22 @@ class RecognitionClass {
       pick_point.z = pick_pose[result.item_id][2];
       pick_point_cloud->push_back(pick_point);
       pcl::transformPointCloud(*pick_point_cloud, *rotated_pick_point_cloud, result.final_transformation);
+      // Use ICP to fine align model to scene
+      t_start = std::clock();
+      pcl::IterativeClosestPoint<pcl::PointXYZ, pcl::PointXYZ> icp;
+      icp.setMaximumIterations (iterations);
+      icp.setInputSource(rotated_model);
+      icp.setInputTarget(scene);
+      pcl::PointCloud<pcl::PointXYZ> Final;
```

Comment:
Please follow variable naming conventions, use camelcase as described in the ros style guide.

Reply:
Will do that.